pkg/kadm/metadata.go

```diff
@@ -227,10 +227,19 @@ func (cl *Client) Metadata(
 	return cl.metadata(ctx, false, topics)
 }
 
+func shouldIncludeClusterAuthorizedOperations(ctx context.Context) bool {
+	if value := ctx.Value(&includeAuthOps); value != nil {
+		if includeTopicAuthorizedOperations, ok := value.(bool); ok {
+			return includeTopicAuthorizedOperations
+		}
```

Comment:
The most minor nit that doesn't need to be pushed -- one thing you could do is to have the value be an empty `struct{}`. The presence of the key is indication enough that the option was opted into. The conditional wrapping this return can be eliminated, too -- if the value exists, then the option was set and we can return true here.